src/tutorial/runtime/introduction.rst

```diff
@@ -168,17 +168,53 @@ Tasks And Jobs
    * Running
    * Succeeded / Failed
 
+The Cylc User Interfaces
+------------------------
+
+.. ifnotslides::
+
+   To help you to keep track of a running workflow Cylc has
+   - A graphical user interface (Cylc GUI).
+   - A text-based user interface (Cylc TUI).
```

Comment:
This hasn't rendered properly
![image](https://user-images.githubusercontent.com/61982285/152023414-1c2651ad-e470-4d21-b7ec-c107a13ce7ce.png)

Reply:
... because the RST is just plain wrong - there should be a space separating the title from the bullets.